src/TrueLayer/MerchantAccounts/MerchantAccountsApi.cs

```diff
@@ -83,8 +83,50 @@ public async Task<ApiResponse<GetPaymentSourcesResponse>> GetPaymentSources(stri
             }
 
             return await _apiClient.GetAsync<GetPaymentSourcesResponse>(
-                _baseUri.Append($"merchant-accounts/{merchantAccountId}/payment-sources?user_id={userId}"),
+                _baseUri.Append($"{merchantAccountId}/payment-sources?user_id={userId}"),
```

Comment:
Did the base URI change? Or, there was an oversight in the previous implementation?

Reply:
it didn't change. The previous implementation was wrong as the `merchant-account` segment of the url was already added [here](https://github.com/TrueLayer/truelayer-dotnet/blob/5c53480cb1e3312cfacf3b032dc6d4332cc77ed7/src/TrueLayer/MerchantAccounts/MerchantAccountsApi.cs#L30)